addons/membership/membership_view.xml

```diff
@@ -234,7 +235,7 @@
                                 <label for="membership_state"/>
                                 <div>
                                     <field name="membership_state"/>
-                                    <button name="%(action_membership_invoice_view)d" type="action" string="Buy Membership" 
+                                    <button name="%(action_membership_invoice_view)d" type="action" string="Buy Membership"
```

Comment:
Yeah, I know, but for keeping the diff clear and reduce possible upstream merge conflicts, we ignore those changes on OCB (not in the rest of the branches).